fluent/src/resolver.js

```diff
@@ -448,6 +448,9 @@ function Pattern(env, ptn) {
   dirty.add(ptn);
   const result = [];
 
+  // Skip isolating if the pattern has only one element.
+  const useIsolating = ctx._useIsolating && ptn.length > 1;
```

Comment:
The comment describes the `false` value while the name of the variable corresponds to the `true` value. Can you update the comment to something like the following please?

    Wrap interpolations with Directional Isolate Formatting characters
    only when the pattern has more than one element.